packages/agent/src/index.ts

```diff
@@ -1,18 +1,25 @@
 export type * from './types/agent.js';
-export type * from './types/managed-key.js';
+export * from './types/dwn.js';
+export type * from './types/identity.js';
+export type * from './types/identity-vault.js';
+export type * from './types/key-manager.js';
+export type * from './types/sync.js';
+export type * from './types/vc.js';
 
-export * from './app-data-store.js';
-export * from './did-manager.js';
-export * from './dwn-manager.js';
-export * from './identity-manager.js';
+export * from './bearer-identity.js';
+export * from './crypto-api.js';
+export * from './did-api.js';
+export * from './dwn-api.js';
+export * from './hd-identity-vault.js';
+export * from './identity-api.js';
 export * from './json-rpc.js';
-export * from './key-manager.js';
-export * from './kms-local.js';
+export * from './local-key-manager.js';
 export * from './rpc-client.js';
-export * from './store-managed-did.js';
-export * from './store-managed-key.js';
-export * from './store-managed-identity.js';
-export * from './sync-manager.js';
-export * from './utils.js';
-
-export * from './test-managed-agent.js';
\ No newline at end of file
+export * from './store-data.js';
+export * from './store-did.js';
+export * from './store-identity.js';
+export * from './store-key.js';
+export * from './sync-api.js';
+export * from './sync-engine-level.js';
+export * from  './test-harness.js';
+export * from './utils.js';
```

Comment:
`DidResolverCacheMemory` is not exported (nor is anything else from `prototyping` folder